src/plugins/home/public/application/components/home.tsx

```diff
@@ -29,6 +29,7 @@ export interface HomeProps {
   localStorage: Storage;
   urlBasePath: string;
   hasUserDataView: () => Promise<boolean>;
+  onSkip?: () => void;
```

Comment:
I think this is not needed anymore :)

Reply:
I confirm it's not needed. It was necessary for the "non-snapshot" tests.